src/System.Text.Primitives/System/Text/Encoding/Utf16/Utf16TextEncoderLE.cs

```diff
@@ -152,6 +152,29 @@ public override bool TryEncode(string text, Span<byte> encodedBytes, out int byt
             return true;
         }
 
+        public override bool TryComputeEncodedBytes(ReadOnlySpan<byte> utf8, out int bytesNeeded)
+        {
+            int characters;
+            var result = Utf8Encoder.TryComputeStringLength(utf8, out characters);
```

Comment:
If we're going c# 7, personally I'd love me some tuples on these Try methods, rather than "returns bool, one or more outs"...